storybooknative/src/Icon/index.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import * as React from "react";
+import { Text as RNText } from "react-native";
+
+import styled from "../styled";
+import defaultTokens from "../defaultTokens";
+import iconsMap from "./icons.json";
+
+import type { Props } from "./index";
+
+const StyledText = styled(RNText, props => {
+  const { theme } = props;
+  return {
+    fontFamily: "orbit-icons",
+    includeFontPadding: false,
+    textAlignVertical: "center",
+    color: theme.orbit.colorIconPrimary,
+    fontSize: 20,
```

Comment:
We should be able to change this

Reply:
`<Icon />` component accepts `style` as a prop, so you'll be able to overwrite default styles.